server/dex/dex.go

```diff
@@ -432,5 +479,65 @@ func NewDEX(cfg *DexConf) (*DEX, error) {
 
 // Config returns the current dex configuration.
 func (dm *DEX) ConfigMsg() json.RawMessage {
-	return dm.config.configEnc
+	dm.configRespMtx.RLock()
+	defer dm.configRespMtx.RUnlock()
+	return dm.configResp.configEnc
+}
+
+func (dm *DEX) MarketRunning(mktName string) (found, running bool) {
+	mkt := dm.markets[mktName]
+	if mkt == nil {
+		return
+	}
+	return true, mkt.Running()
```

Comment:
Similar to `OrderBook` being able to track running markets, `DEX` can also determine which markets are running based on which market subsystems exist and have returned.

Reply:
To be clear, you mean `OrderRouter` here, right?

Reply:
Yup, `OrderRouter` is what I meant to say.